repepo/paper/make_figures_rebuttals.py

```
@@ -0,0 +1,239 @@
+# %%
+import pandas as pd
+import seaborn as sns
+import json
+import matplotlib.pyplot as plt
+from sklearn.calibration import column_or_1d
```

Ruff

`sklearn.calibration.column_or_1d` imported but unused

Remove unused import: `sklearn.calibration.column_or_1d`

(F401)